README.md

```diff
@@ -84,16 +82,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      # Used to host cibuildwheel
-      - uses: actions/setup-python@v2
-
       - name: Install cibuildwheel
         run: python -m pip install cibuildwheel==1.8.0
```

Comment:
I would leave the main example default and not the action to it can adapt most easily to other CI's; I would also not remove the setup-python step if using pip, only if using pipx.

Reply:
It is invalid to use `python` without using setup-python first; this will give you Python 2 on macOS. And `python3` is being added to Windows at our request, but for now, `python3 ...` will fail on Windows. In general, pipx is safe to use in the default system, but pip should always come from setup-python.

Reply:
You _could_ change to using pipx, but that wouldn't be as applicable to other CI's, except for Azure, probably.

Reply:
thanks @henryiii, that's very helpful

Reply:
i'll restore `setup-python` in these examples.